### PR TITLE
5 create a customer model

### PR DESCRIPTION
**Pull request by ShalaevPavel**
Added migrations and model classes
Run rails db:migrate

### CODE REVIEW

File: app/models/adress.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Adress< ActiveRecord::Base
+
```
Comment (Ikl0) [inline app/models/adress.rb:3]: associations are not implemented yet? or they are not needed in this models?

File: db/migrate/20230713112745_create_customers.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class CreateCustomers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :customers do |t|
+      t.string :email
```
Comment (Ikl0) [inline db/migrate/20230713112745_create_customers.rb:5]: I think you forgot about shopify_id(customer id from shopify)

File: db/migrate/20230713112750_create_addresses.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class CreateAddresses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :addresses do |t|
+      t.integer :customer_id
```
Comment (Ikl0) [inline db/migrate/20230713112750_create_addresses.rb:4]: same as customer(shopify_id)

**Review by Ikl0 — CHANGES_REQUESTED** [review]

**Comment by Ikl0** [comment]
Need to understand where models require if from shopify

File: db/migrate/20230713112745_create_customers.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      t.string :state
+      t.integer :last_order_id
+      t.string :last_order_name
+      t.integer :orders_count
+      t.string :total_spent
+      t.boolean :tax_exempt
+      t.string :multipass_identifier
+      t.string :note
+      t.string :password
+      t.string :password_confirmation
```
Comment (fooooxmr) [inline db/migrate/20230713112745_create_customers.rb]: I do not think, that we will store password in our Database.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

**Review by Ikl0 — APPROVED** [review]

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**